src/metabase/api/setup.clj

```diff
@@ -115,9 +116,9 @@
         num-tables         (db/count 'Table)
         num-cards          (db/count 'Card)
         num-users          (db/count 'User)]
-    [{:title       "Add a database"
-      :group       "Get connected"
-      :description "Connect to your data so your whole team can start to explore."
+    [{:title       (trs "Add a database")
```

Comment:
It looks like this is getting presented to the user. If that's the case, you want `tru` not `trs`. `trs` will return text in the system's local, not the (browser) user's locale.

Reply:
👍